micromamba/src/run.cpp

```diff
@@ -389,10 +389,11 @@ set_run_command(CLI::App* subcom)
 
             // Make sure the proc directory is always existing and ready.
             std::error_code ec;
-            fs::create_directories(proc_dir(), ec);
-            if (!ec)
+            bool is_created = fs::create_directories(proc_dir(), ec);
+            if (!is_created && ec)
```

Comment:
It works but wh not simply ```if (ec)``` ? Can `create_directories` succeed but still return a non zero error code?

Reply:
I am not sure what happens when the directory already exists (the manual says that it returns `true` if directories are created). But yeah, checking only `ec` would probably be enough.